### PR TITLE
Graphrbac description update

### PR DESCRIPTION
**Pull request by mmacy**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [ ] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/723)

### CODE REVIEW

**Comment by mmacy** [comment]
@amarzavery Please do not merge yet. Checking whether this will build.

File: arm-graphrbac/1.6/swagger/graphrbac.json [hunk 1]
```diff
@@ -111,14 +111,14 @@
           "Application"
         ],
         "operationId": "Applications_List",
-        "description": "Lists applications by filter parameters. Reference: http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx",
+        "description": "Lists applications by filter parameters. For more information, see http://msdn.microsoft.com/library/azure/hh974476.aspx.",
```
Comment (mmacy) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: @erickson-doug There are MSDN links like this in most of the operation descriptions. Keep or delete?

File: arm-graphrbac/1.6/swagger/graphrbac.json [hunk 2]
```diff
@@ -270,14 +270,14 @@
           "ApplicationKeyCredentials"
         ],
         "operationId": "Applications_ListKeyCredentials",
-        "description": "Get keyCredentials associated with the application by object Id. Reference: https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/entity-and-complex-type-reference#keycredential-type",
+        "description": "Get the keyCredentials associated with an application.",
```
Comment (erickson-doug) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: Can we capture what these key credentials (keyCredentials) are in a second sentence? This update can wait until after the 16th, though.

Comment (mmacy) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: Yes, there are quite a few other opportunities for improvement with similar format. Could do for another pass, for sure.

File: arm-graphrbac/1.6/swagger/graphrbac.json [hunk 3]
```diff
@@ -327,7 +327,7 @@
             "schema": {
               "$ref": "#/definitions/KeyCredentialsUpdateParameters"
             },
-            "description": "Parameters to update keyCredentials of an existing application."
+            "description": "Parameters to update the keyCredentials of an existing application."
```
Comment (erickson-doug) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: I'm guess keyCredentials are the Azure AD client key, the passwordCredential is the client password, etc? We should probably clarify that.

File: arm-graphrbac/1.6/swagger/graphrbac.json [hunk 4]
```diff
@@ -1224,14 +1224,14 @@
           "User"
         ],
         "operationId": "Users_Get",
-        "description": "Gets user information from the directory. Reference: https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/users-operations#GetAUser",
+        "description": "Gets user information from the directory.",
```
Comment (erickson-doug) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: Might wanna spell out "Azure Active Directory" here?

Comment (mmacy) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: As discussed, staying the course on this one as well--no change.

**Pull request merged**